{"uuid":"f35b7705-46f8-4215-8f1f-4b8b21ef6774","historyId":"6911cc5dc8b191c370352720f18f7e38","fullName":"tests.IncorrectLogin.incorrectLoginTest","labels":[{"name":"package","value":"tests.IncorrectLogin"},{"name":"testClass","value":"tests.IncorrectLogin"},{"name":"testMethod","value":"incorrectLoginTest"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"tests.IncorrectLogin"},{"name":"host","value":"LAPTOP-7B3LLLRV"},{"name":"thread","value":"29364@LAPTOP-7B3LLLRV.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"normal"}],"links":[],"name":"Utilizar credenciales no válidas","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h3[contains(text(),'Epic sadface: Username and password do not match any user in this service')]' (tried for 60 second(s) with 500 milliseconds interval)","trace":"java.lang.AssertionError: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h3[contains(text(),'Epic sadface: Username and password do not match any user in this service')]' (tried for 60 second(s) with 500 milliseconds interval)\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat core.PageBase.waitForVisibility(PageBase.java:43)\r\n\tat core.PageBase.compareText(PageBase.java:80)\r\n\tat pages.LoginPage.wrongLogin(LoginPage.java:37)\r\n\tat tests.IncorrectLogin.incorrectLoginTest(IncorrectLogin.java:35)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","description":"Introducir credenciales no validas en la página de login","steps":[{"name":"ingresando usuario y contraseña","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h3[contains(text(),'Epic sadface: Username and password do not match any user in this service')]' (tried for 60 second(s) with 500 milliseconds interval)","trace":"java.lang.AssertionError: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //h3[contains(text(),'Epic sadface: Username and password do not match any user in this service')]' (tried for 60 second(s) with 500 milliseconds interval)\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat core.PageBase.waitForVisibility(PageBase.java:43)\r\n\tat core.PageBase.compareText(PageBase.java:80)\r\n\tat pages.LoginPage.wrongLogin(LoginPage.java:37)\r\n\tat tests.IncorrectLogin.incorrectLoginTest(IncorrectLogin.java:35)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"customer","value":"Customer(incorrectUser=standard_user, incorrectPassword=secret_sauce, customerName=Albert, customerLastname=Thomas, zipCode=1111)"}],"start":1674597937979,"stop":1674597998800}],"attachments":[{"name":"page screenshot","source":"c7b1bda5-c047-4ae5-a51f-06e96d5240b9-attachment","type":"image/png"},{"name":"Browser Log","source":"0df61692-108a-4fa2-926f-c2361f2056ac-attachment","type":"text/plain"},{"name":"Exception_Log","source":"4c22fc63-9bbb-4b29-a2a5-c7e29f8b3509-attachment","type":"text/plain"}],"parameters":[],"start":1674597936136,"stop":1674597999294}